client/src/components/Create-patient-symptom.component.js

```diff
@@ -1,233 +1,294 @@
+    this.onCheckedSymptom2 = this.onCheckedSymptom2.bind(this);
+    this.onCheckedSymptom3 = this.onCheckedSymptom3.bind(this);
+    this.onCheckedSymptom4 = this.onCheckedSymptom4.bind(this);
+    this.onChangeTemperature = this.onChangeTemperature.bind(this);
+    this.onMediaChange = this.onMediaChange.bind(this);
+    this.onChangeadditionalNote = this.onChangeadditionalNote.bind(this);
 
-    this.state ={
+    this.onChangeAttention = this.onChangeAttention.bind(this)
+    this.onSubmit = this.onSubmit.bind(this);
+    // preserve the initial state in a new object
+    this.baseState = this.state
+    this.state = {
       date: Date().toLocaleString(),
       symptom1: '',
       symptom2: '',
       symptom3: '',
       symptom4: '',
-      additionalNote: '',
-      media: ''
-      // patientName
-      //patientID
-
-
+      temperature: '',
+      mediaDescription: '',
+      media: null,
+      attention: false,// onclick enable alert lable 
+      additionalNote: ''
     
-     // this.fileInput = React.createRef();
-   
     }
- 
+
+
   }
 
- /* onChangeDate(e)
-  {
-    this.setState ({
-      date:e.target.value
-      
+  onChangeDate(e) {
+    this.setState({
+      date: e.target.value
+
     });
-  } */
-  onCheckedSymptom1(e) 
-  {
+  };
+  componentDidMount() {
+
     this.setState({
-      sympetom1:e.target.value
-      
+
+      provider: ['test test'],
+
+      doctorNote: 'test Note'
+
     });
+
   }
-  onCheckedSymptom2(e)
-  {
+
+  onChangeAttention(e) {
     this.setState({
-      symptom2:e.target.value
-      
+      attention: e.target.value
+
     });
-  }
-  onCheckedSymptom3(e)
-  {
+  };
+  //on file select 
+
+  onMediaChange = event => {
+
+    // Update the state 
+    this.setState({ media: event.target.files[0] });
+
+  };
+
+  //On file upload (click upload button)
+  onFileUpload(e) {
+    //create an object of form data
+    const patientMedia = new FormData();
+    //update the formData object
+    patientMedia.append(
+      "myFile",
+      this.state.media,
+      this.state.media.name
+    );
+    //file upload info
+    //console.log(this.state.media);
+    axios.post("http://localhost:3000/uploadfile", patientMedia);
+
+  };
+  onCheckedSymptom1(e) {
     this.setState({
-      symptom3:e.target.checked
-      
+      sympetom1: e.target.value
+
     });
-  }
-  onCheckedSymptom4(e)
-  {
+  };
+
+  onCheckedSymptom2(e) {
     this.setState({
-      symptom4:e.target.checked
-      
+      symptom2: e.target.value
+
     });
-  }
-  onChangeadditionalNote(e)
-  {
+  };
+  onCheckedSymptom3(e) {
     this.setState({
-      additionalNote:e.target.value
-      
+      symptom3: e.target.checked
+
     });
-  }
-  onChangeMedia(e)
-  {
+  };
+  onCheckedSymptom4(e) {
+    this.setState({
+      symptom4: e.target.checked
+
+    });
+  };
+  onChangeadditionalNote(e) {
     this.setState({
-      media:e.target.value
-      
+      additionalNote: e.target.value

```

Comment:
instead of setting the state as a "target.value", making it "true" might be better. Patients should not type true, or false, they should just click a button that says "immediate attention" and that button when clicked, should make the state true.